WcaOnRails/app/views/competitions/show_events.html.erb

```diff
@@ -0,0 +1,68 @@
+<% provide(:title, @competition.name) %>
```

Comment:
Yes, this is meant to be a page like this one: https://euro2016.cubing.net/events.

Nah, I plan to add a edit ui in a later pr (it's gonna be javascripty, and I thought I'd get better feedback on the database schema if I kept this pr simple)